src/main/java/com/spotify/reaper/cassandra/JmxProxy.java

```diff
@@ -287,9 +293,8 @@ public int getPendingCompactions() {
   public boolean isRepairRunning() {
     // Check if AntiEntropySession is actually running on the node
     try {
-      ObjectName name = new ObjectName(AES_OBJECT_NAME);
-      int activeCount = (Integer) mbeanServer.getAttribute(name, "ActiveCount");
-      long pendingCount = (Long) mbeanServer.getAttribute(name, "PendingTasks");
+      int activeCount = (Integer) mbeanServer.getAttribute(new ObjectName(AES_ACTIVE_OBJECT_NAME), "Value") + (Integer) mbeanServer.getAttribute(new ObjectName(VALIDATION_ACTIVE_OBJECT_NAME), "Value");
+      long pendingCount = (Long) mbeanServer.getAttribute(new ObjectName(AES_PENDING_OBJECT_NAME), "Value") + (Long) mbeanServer.getAttribute(new ObjectName(VALIDATION_PENDING_OBJECT_NAME), "Value");;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)